src/plot_api/plot_api.js

```diff
@@ -277,6 +277,7 @@ function _doPlot(gd, data, layout, config) {
 
         subroutines.drawMarginPushers(gd);
         Axes.allowAutoMargin(gd);
+        Plots.allowAutoMargin(gd, 'title.automargin');
```

Comment:
Shouldn't we ensure there is a title text and automargin before calling this?
```suggestion
        if(gd._fullLayout.title.text && gd._fullLayout.title.automargin) Plots.allowAutoMargin(gd, 'title.automargin');
```